csrc/scheduler/utils.h

```diff
@@ -649,6 +649,17 @@ DisjointSets<IterDomain*> disjointLogicalSets(Fusion* fusion);
 // [1, 0, 0] pos 1 would return true
 bool breakIsDisjoint(std::vector<int64_t> group_ids, int64_t pos);
 
+// Transform the ids_to_transform as progressing through the transform_exprs
+void applyTransforms(
+    std::vector<IterDomain*>& ids_to_transform,
+    const std::vector<Expr*>& transform_exprs);
+
+// Returns a mapping from vec1 to vec2.
```

Comment:
Does this assume vec2 must be a permutation of vec1? This is an important assumption that's worth code-commenting.